etc/kayobe/inventory/group_vars/seed/docker-registry

```diff
@@ -0,0 +1,7 @@
+---
+###############################################################################
+# Seed node docker regsitry configuration.
+
+# Whether to attempt a basic authentication login to a registry when
+# deploying seed containers
+deploy_containers_registry_attempt_login: False
```

Comment:
There is `seed_deploy_containers_registry_attempt_login` which is probably more appropriate, and can be set in seed.yml.

Reply:
Could set it to `{{ not seed_pulp_container_enabled | bool }}`?

Reply:
Done